### DISCUSSION TITLE
Download Just Subtitles?

### DISCUSSION THREAD

**Discussion by transmatrix** [crunchy-labs/crunchy-cli#354] [Q&A]
Was recently forced to get One Piece episodes from Crunchyroll instead of where I was getting them before: Funimation. Crunchyroll English dubs are missing most/all forced subtitles, though. They're present on the Japanese language video, but not on the English language videos. Right now, I'm downloading both versions of the video so that I can strip the subtitles off of the Japanese language video and put them on the English language video. However, it would be much simpler if I could just download the subtitles and not the whole video. Is this possible? If not currently, could you add this option to the downloader? Feel like I'm wasting bandwidth (and it takes longer to download.)

**Comment by Sylar40** [comment]
I would also like this to work. It's an alternative to solving the problem, but I would really like to have ONLY the subtitles

**Comment by BattouKen** [comment]
Pues sí la verdad es que falta la opción de video como las de audio y subtítulo uno de video para poder seleccionar tipo así 

`-a None -s es-419 –video None`
